apps/aecontract/src/aect_contracts.erl

```diff
@@ -141,7 +141,8 @@ is_legal_version_in_protocol(call, #{vm := VMVersion}, ProtocolVersion) ->
         ?ROMA_PROTOCOL_VSN    when VMVersion =:= ?VM_AEVM_SOPHIA_1 -> true;
         ?MINERVA_PROTOCOL_VSN when VMVersion =:= ?VM_AEVM_SOPHIA_1;
                                    VMVersion =:= ?VM_AEVM_SOPHIA_2 -> true;
-        _                     when VMVersion =:= ?VM_AEVM_SOLIDITY_1 -> ?VM_AEVM_SOLIDITY_1_enabled
+        _                     when VMVersion =:= ?VM_AEVM_SOLIDITY_1 -> ?VM_AEVM_SOLIDITY_1_enabled;
+        _ -> false
```

Comment:
Without this clause - when an unexpexted protocol & version are received - function crashes bad